README.md

```diff
@@ -448,15 +448,15 @@ Then use the `jsonGraphqlExpress` express middleware:
 
 ```js
 import express from 'express';
-import jsonGraphqlExpress from 'json-graphql-server';
+import jsonGraphqlExpress from 'json-graphql-server/node';
```

Comment:
```suggestion
const { default: jsonGraphqlExpress } = require("json-graphql-server/node");
```

Reply:
I reverted the change, the doc was right initially